server/grpc_service.go

```diff
@@ -2316,6 +2319,9 @@ func (s *GrpcServer) StoreGlobalConfig(_ context.Context, request *pdpb.StoreGlo
 // - `Names` iteratively get value from `ConfigPath/Name` but not care about revision
 // - `ConfigPath` if `Names` is nil can get all values and revision of current path
 func (s *GrpcServer) LoadGlobalConfig(ctx context.Context, request *pdpb.LoadGlobalConfigRequest) (*pdpb.LoadGlobalConfigResponse, error) {
+	if s.client == nil {
```

Comment:
does it only happen when the server is starting and no start etcd server?

Reply:
As far as I know, this should be the only case

Reply:
Why does this case happen?

Reply:
When grpc comes in before etcd has finished booting.